compact/far_forward/ZDC.xml

```diff
@@ -1,6 +1,3 @@
-<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
```

Comment:
Don't remove the copyright headers.

Reply:
The copyright headers have been put back into the ZDC.xml.